ui/editor/src/ctrl.ts

```diff
@@ -190,6 +190,11 @@ export default class EditorCtrl {
     return `${this.cfg.baseUrl}/${urlFen(fen)}${variant}${orientationParam}`;
   }
 
+  makeImageUrl(fen: string, orientation: Color = 'white'): string {
+    const url = `https://lichess1.org/export/fen.gif?fen=${fen}&color=${orientation}`;
```

Comment:
Can you change the URL so that `lichess1.org` is not hardcoded in? If you go to a game and the Share tab, the links there will work with what is set in the config.

![image](https://github.com/lichess-org/lila/assets/271432/448fa1ad-9afe-4226-a441-c55ab9e14ba1)